### PR TITLE
RSDK-2062 Document build and run steps

### PR DESCRIPTION
**Pull request by maximpertsov**
JIRA: https://viam.atlassian.net/browse/RSDK-2062

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+### 1. Build binary
+
+If you clone this repository to the target environment where you run your Viam robot, then you can build a binary named `wifi` with:
+
+```
+go build -o wifi
+```
+
+Alternatively, if you want to build this a binary for a different target environment, please use the [viam canon tool](https://github.com/viamrobotics/canon).
```
Comment (Otterverse) [inline README.md:15]:
```suggestion
Alternatively, if you want to build a binary for a different target environment, please use the [viam canon tool](https://github.com/viamrobotics/canon).
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+
+### 2. Add to robot configuration
+
+Copy the binary to in the environment where your Viam robot is running and add the following to your configuration:
```
Comment (Otterverse) [inline README.md:19]:
```suggestion
Copy the binary to the robot (system where viam-server is running) and add the following to your configuration:
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+  ...
+  "modules": [
+    ...,
+    {
+      "executable_path": "<path_to_binary>",
+      "name": "wifi"
+    }
+    ...,
+  ],
+  ...
```
Comment (Otterverse) [inline README.md:31]: I would also add the config for the wifi unit itself here. The part that would go in the "components" section.

**Review by Otterverse — CHANGES_REQUESTED** [review]
Couple of small nits, but I also think you really need to list how to configure the component itself, not just the module.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,45 @@
...
+    ...,
+  ],
+  "components": [
+    ...,
+    {
+      "name": "wifi",
+      "type": "sensor",
+      "model": "viam-labs:sensor:linux-wifi",
+      "depends_on": [],
+      "attributes": {}
```
Comment (Otterverse) [inline README.md:38]: [nit] I think these could just be left out entirely, but no harm with them here either.

**Review by Otterverse — APPROVED** [review]
LGTM

File: README.md (hunk 4 above)
Comment (maximpertsov) [inline README.md:38]:
```suggestion
      "model": "viam-labs:sensor:linux-wifi"
```

**Pull request merged**